pkg/build/api/types.go

```diff
@@ -42,8 +49,8 @@ type Build struct {
 // BuildSpec encapsulates all the inputs necessary to represent a build.
 type BuildSpec struct {
 	// ServiceAccount is the name of the ServiceAccount to use to run the pod
-	// created by this build.
-	// The pod will be allowed to use secrets referenced by the ServiceAccount
+	// created by this build. The pod will be allowed to use secrets referenced
```

Comment:
@mfojtik you didn't typo, i did. wrote 69 first then i corrected :))

@bparees  so i "adopted" this from here:  https://github.com/golang/go/wiki/CodeReviewComments#line-length 
but i can follow up with how it's done in other files. if there isn't maybe we can make it as a standard? or put it up for debate as a standard, either way.
For now, what i will do is remove the doc changes, and will submit a PR or not based on what the other teams use and what i find out. I'm guessing that would ease the work of reviewing this PR, i'm not trying to sabotage myself :-)